ports/mimxrt10xx/Makefile

```diff
@@ -88,6 +88,7 @@ ifeq ($(DEBUG), 1)
   # You may want to enable these flags to make setting breakpoints easier.
   CFLAGS += -fno-inline -fno-ipa-sra
 else
+  CFLAGS += -ggdb
```

Comment:
This doesn't seem right

Reply:
Didn't somebody run a test the other day and find that the increase in flash size for builds with --ggdb on was under a kilobyte? If that's wrong I'll take this out

Reply:
Here's the Issue: #2790

Reply:
Of course, it shouldn't be in the ifeq if it's in both cases... I'll move it.

Reply:
I see, wasn't aware, just felt a bit like a random change and having it on both sides of the ifdef made it ever more confusing :D